### DISCUSSION TITLE
when an mergify backport a new pr, how can link this new pr to the original issue

### DISCUSSION THREAD

**Discussion by dulong41** [Mergifyio/mergify#5059] [Q&A]
When an issue creates, then a pull request will be created to resolve this issue. 
while when a mergify bot backport some new pull requests, the new created pull request does not inherit the original pull request linked issue info. for the mergify bot created pr, usually the users does not have permissions to modify the commit message, thus, i'm wondering whether the mergify could backport the new pr with the linked issues? use the command like: @mergify backport new-branch #253

**Comment by jd** [comment]
The commit message are kept on a backport, so I imagine you're referring to the PR body.
You [can configure the body of a PR created by a backport](https://docs.mergify.com/actions/backport/#options
), but you cannot make it include/copy the original PR content, unfortunately (yet?).
